### ISSUE TITLE
URL shortening website using Python

### ISSUE THREAD

**Issue by ajruttam** [TusharKesarwani/Front-End-Projects#19]
Please assign this to me @TusharKesarwani

**Comment by Diptamoy-Mitra** [comment]
@TusharKesarwani  please assign me this

**Comment by Vanshiii203** [comment]
@ajruttam Are you still working on this issue?

**Comment by ajruttam** [comment]
@Vanshiii203 yes, I am kinda busy with my college stuff so unable to work on this issue well. But yea, i am working on this

**Comment by Vanshiii203** [comment]
> @Vanshiii203 yes, I am kinda busy with my college stuff so unable to work on this issue well. But yea, i am working on this

Okay
